### PR TITLE
Using More Cupertino Widgets

### PR DESCRIPTION
**Pull request by MapleSyyrup**
- Fix the chart in IOS using the safe area widget.
- Add more cupertino widgets for IOS.
- Create another file for adaptive cupertino buttons to make it more
  easier to manage.

image
image
image

### CODE REVIEW

File: lib/widgets/adaptive_flat_button.dart [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                        : FlatButton(
+                            textColor: Theme.of(context).primaryColor,
+                            child: Text(
+                              text,
+                              style: TextStyle(fontWeight: FontWeight.bold),
+                            ),
+                            onPressed: handler,
+                          );
+  }
+}
```
Comment (Nei1eveN) [inline lib/widgets/adaptive_flat_button.dart]: add new line at the end of file

File: lib/widgets/new_transaction.dart [hunk 2]
```diff
@@ -90,14 +92,7 @@ class _NewTransactionState extends State<NewTransaction> {
...
                     ),
-                    FlatButton(
-                      textColor: Theme.of(context).primaryColor,
-                      child: Text(
-                        'Choose Date',
-                        style: TextStyle(fontWeight: FontWeight.bold),
-                      ),
-                      onPressed: _presentDatePicker,
-                    ),
+                    AdaptiveFlatButton('Choose Date', _presentDatePicker)
```
Comment (Nei1eveN) [inline lib/widgets/new_transaction.dart]: trailing commas

File: lib/widgets/adaptive_flat_button.dart (hunk 1 above)
Comment (Nei1eveN) [inline lib/widgets/adaptive_flat_button.dart:31]: @MapleSyyrup this is still wrong. add a new line that does not have this closing bracket

Comment (Nei1eveN) [reply lib/widgets/adaptive_flat_button.dart]: unresolving this. see this comment: https://github.com/MapleSyyrup/Personal-Expenses/pull/12#discussion_r533463164

Comment (MapleSyyrup) [reply lib/widgets/adaptive_flat_button.dart:31]: > @MapleSyyrup this is still wrong. add a new line that does not have this closing bracket

@Nei1eveN a blank line was added
image

Comment (Nei1eveN) [reply lib/widgets/adaptive_flat_button.dart:31]: @MapleSyyrup not the way I see it
image

Comment (Nei1eveN) [reply lib/widgets/adaptive_flat_button.dart:31]: @MapleSyyrup make sure to apply a blank line on your new commit

**Pull request merged**